core/src/main/scala/org/apache/spark/scheduler/Task.scala

```diff
@@ -87,12 +87,19 @@ private[spark] abstract class Task[T](val stageId: Int, var partitionId: Int) ex
   // initialized when kill() is invoked.
   @volatile @transient private var _killed = false
 
+  protected var _executorDeserializeTime: Long = 0
+
   /**
    * Whether the task has been killed.
    */
   def killed: Boolean = _killed
 
   /**
+   * Returns the amount of time spent on task / RDD deserialization.
```

Comment:
maybe "spent deserializing the RDD and function to be run"? (since technically deserialization of the task happens in Executor.scala, not here)